github/GithubIntegration.py

```diff
@@ -0,0 +1,140 @@
+import time
```

Comment:
Moved from `MainClass`, it's already big as it is and that does not really belong there.